go/store/nbs/chunk_source_adapter.go

```diff
@@ -53,3 +54,7 @@ func (csa chunkSourceAdapter) clone() (chunkSource, error) {
 	}
 	return &chunkSourceAdapter{tr, csa.h}, nil
 }
+
+func (csa chunkSourceAdapter) getAllChunkHashes(_ context.Context, _ chan<- hash.Hash, _ *sync.WaitGroup) int {
+	panic("unimplemented")
+}
```

Comment:
Is this the right implementation? Why can't we use the index in the table reader to forward this along?

Reply:
In the chunkSourceAdapter, you have access to a chunkReader, but the iterateAllChunks is on the chunkSource. I could change this, I suppose. Maybe I need to understand what a chunkSourceAdapter is to have an opinion.